community/examples/gke.yaml

```diff
@@ -43,3 +43,12 @@ deployment_groups:
   - id: compute_pool
     source: community/modules/compute/gke-node-pool
     use: [gke_cluster]
+
+  - id: job-template
+    source: community/modules/compute/gke-job-template
+    use: [compute_pool]
+    settings:
+      image: busybox
+      command: [echo, Hello World]
```

Comment:
here, just once, I will say that maybe it would be nice to itemize this list. It took me a second look to understand you are passing two items to this command, and then remember our cloud builds.

Reply:
Done.